client/state/ui/action-log/reducer.js

```diff
@@ -16,7 +16,12 @@ import {
 	SITE_SETTINGS_RECEIVE,
 } from 'state/action-types';
 
+const relevantAnalyticsEvents = [
+	'calypso_themeshowcase_theme_click',
+];
+
 const relevantTypes = {
+	ANALYTICS_EVENT_RECORD: isRelevantAnalytics,
```

Comment:
It's weird to have this here and not import the action type, yet if I do import it eslint will rightfully complain that I don't use it. Question: do we prefer to keep this as-is, or do we prefer an explicit import with a `no-unused-vars` exception?